### ISSUE TITLE
Littlefs example does not generate the storage partition (IDFGH-13205)

### ISSUE THREAD

**Issue by doragasu** [espressif/esp-idf#14142]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### IDF version.

v5.2.2

### Operating System used.

Linux

### How did you build your project?

Command line with idf.py

### If you are using Windows, please specify command line type.

None

### What is the expected behavior?

I copy the `examples/storage/littlefs` and build it with `idf.py build`. The build does not error, but when finished, the `build/storage.bin` file with the littlefs filesystem is not generated. So `idf.py flash` fails because the file is not found.

### What is the actual behavior?

During build, the littlefs python is not called to generate the partition, but no error is shown:
```Bash
$ idf.py build
Executing action: all (aliases: build)
Running ninja in directory /home/esp/src/github/testing/littlefs/build
Executing "ninja all"...
[11/971] Generating ../../partition_table/partition-table.bin
Partition table binary generated. Contents:
*******************************************************************************
# ESP-IDF Partition Table
# Name, Type, SubType, Offset, Size, Flags
nvs,data,nvs,0x9000,24K,
phy_init,data,phy,0xf000,4K,
factory,app,factory,0x10000,1M,
storage,data,littlefs,0x110000,960K,
*******************************************************************************
[394/971] Generating ../../littlefs_py_venv
Ignoring littlefs-python: markers 'python_version < "3.8"' don't match your environment
Collecting littlefs-python>=0.7.1
  Downloading littlefs_python-0.11.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (982 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 982.9/982.9 kB 19.2 MB/s eta 0:00:00
Installing collected packages: littlefs-python
Successfully installed littlefs-python-0.11.0
[416/971] Performing configure step for 'bootloader'
-- Found Git: /usr/bin/git (found version "2.39.2") 
-- The C compiler identification is GNU 13.2.0
-- The CXX compiler identification is GNU 13.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/esp/.espressif/tools/xtensa-esp-elf/esp-13.2.0_20230928/xtensa-esp-elf/bin/xtensa-esp32s3-elf-gcc
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/esp/.espressif/tools/xtensa-esp-elf/esp-13.2.0_20230928/xtensa-esp-elf/bin/xtensa-esp32s3-elf-gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/esp/.espressif/tools/xtensa-esp-elf/esp-13.2.0_20230928/xtensa-esp-elf/bin/xtensa-esp32s3-elf-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Building ESP-IDF components for target esp32s3
-- Project sdkconfig file /home/esp/src/github/testing/littlefs/sdkconfig
-- Compiler supported targets: xtensa-esp-elf
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of time_t
-- Check size of time_t - done
-- Adding linker script /home/esp/dev/esp-idf/components/soc/esp32s3/ld/esp32s3.peripherals.ld
-- Bootloader project name: "bootloader" version: 1
-- Adding linker script /home/esp/dev/esp-idf/components/esp_rom/esp32s3/ld/esp32s3.rom.ld
-- Adding linker script /home/esp/dev/esp-idf/components/esp_rom/esp32s3/ld/esp32s3.rom.api.ld
-- Adding linker script /home/esp/dev/esp-idf/components/esp_rom/esp32s3/ld/esp32s3.rom.libgcc.ld
-- Adding linker script /home/esp/dev/esp-idf/components/esp_rom/esp32s3/ld/esp32s3.rom.newlib.ld
-- Adding linker script /home/esp/dev/esp-idf/components/bootloader/subproject/main/ld/esp32s3/bootloader.ld
-- Adding linker script /home/esp/dev/esp-idf/components/bootloader/subproject/main/ld/esp32s3/bootloader.rom.ld
-- Components: bootloader bootloader_support efuse esp_app_format esp_bootloader_format esp_common esp_hw_support esp_rom esp_system esptool_py freertos hal log main micro-ecc newlib partition_table soc spi_flash xtensa
-- Component paths: /home/esp/dev/esp-idf/components/bootloader /home/esp/dev/esp-idf/components/bootloader_support /home/esp/dev/esp-idf/components/efuse /home/esp/dev/esp-idf/components/esp_app_format /home/esp/dev/esp-idf/components/esp_bootloader_format /home/esp/dev/esp-idf/components/esp_common /home/esp/dev/esp-idf/components/esp_hw_support /home/esp/dev/esp-idf/components/esp_rom /home/esp/dev/esp-idf/components/esp_system /home/esp/dev/esp-idf/components/esptool_py /home/esp/dev/esp-idf/components/freertos /home/esp/dev/esp-idf/components/hal /home/esp/dev/esp-idf/components/log /home/esp/dev/esp-idf/components/bootloader/subproject/main /home/esp/dev/esp-idf/components/bootloader/subproject/components/micro-ecc /home/esp/dev/esp-idf/components/newlib /home/esp/dev/esp-idf/components/partition_table /home/esp/dev/esp-idf/components/soc /home/esp/dev/esp-idf/components/spi_flash /home/esp/dev/esp-idf/components/xtensa
-- Configuring done
-- Generating done
-- Build files have been written to: /home/esp/src/github/testing/littlefs/build/bootloader
[111/112] Generating binary image from built executable
esptool.py v4.7.0
Creating esp32s3 image...
Merged 2 ELF sections
Successfully created esp32s3 image.
Generated /home/esp/src/github/testing/littlefs/build/bootloader/bootloader.bin
[112/112] cd /home/esp/src/github/testing/littlefs/build/bootloader/esp-idf/esptool_py && /home/esp/.esp....py --offset 0x8000 bootloader 0x0 /home/esp/src/github/testing/littlefs/build/bootloader/bootloader.bin
Bootloader binary size 0x5250 bytes. 0x2db0 bytes (36%) free.
[970/971] Generating binary image from built executable
esptool.py v4.7.0
Creating esp32s3 image...
Merged 2 ELF sections
Successfully created esp32s3 image.
Generated /home/esp/src/github/testing/littlefs/build/littlefs_example.bin
[971/971] cd /home/esp/src/github/testing/littlefs/build/esp-idf/esptool_py && /home/esp/.espressif/pyth...ild/partition_table/partition-table.bin /home/esp/src/github/testing/littlefs/build/littlefs_example.bin
littlefs_example.bin binary size 0x410e0 bytes. Smallest app partition is 0x100000 bytes. 0xbef20 bytes (75%) free.

Project build complete. To flash, run:
 idf.py flash
or
 idf.py -p PORT flash
or
 python -m esptool --chip esp32s3 -b 460800 --before default_reset --after hard_reset write_flash --flash_mode dio --flash_size 2MB --flash_freq 80m 0x0 build/bootloader/bootloader.bin 0x8000 build/partition_table/partition-table.bin 0x10000 build/littlefs_example.bin 0x110000 build/storage.bin
or from the "/home/esp/src/github/testing/littlefs/build" directory
 python -m esptool --chip esp32s3 -b 460800 --before default_reset --after hard_reset write_flash "@flash_args"
```
But as `storage.bin` file is not generated, trying to flash the project fails:

```Bash
$ idf.py flash
Executing action: flash
Serial port /dev/ttyACM0
Connecting...
Detecting chip type... ESP32-S3
Running ninja in directory /home/esp/src/github/testing/littlefs/build
Executing "ninja flash"...
[1/5] cd /home/esp/src/github/testing/littlefs/build/esp-idf/esptool_py && /home/esp/.espressif/python_e...ild/partition_table/partition-table.bin /home/esp/src/github/testing/littlefs/build/littlefs_example.bin
littlefs_example.bin binary size 0x410e0 bytes. Smallest app partition is 0x100000 bytes. 0xbef20 bytes (75%) free.
[1/1] cd /home/esp/src/github/testing/littlefs/build/bootloader/esp-idf/esptool_py && /home/esp/.espress....py --offset 0x8000 bootloader 0x0 /home/esp/src/github/testing/littlefs/build/bootloader/bootloader.bin
Bootloader binary size 0x5250 bytes. 0x2db0 bytes (36%) free.
[4/5] cd /home/esp/dev/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/esp/dev/esp-idf...p/src/github/testing/littlefs/build -P /home/esp/dev/esp-idf/components/esptool_py/run_serial_tool.cmakeusage: esptool write_flash [-h] [--erase-all]
                           [--flash_freq {keep,80m,60m,48m,40m,30m,26m,24m,20m,16m,15m,12m}]
                           [--flash_mode {keep,qio,qout,dio,dout}]
                           [--flash_size {detect,keep,256KB,512KB,1MB,2MB,2MB-c1,4MB,4MB-c1,8MB,16MB,32MB,64MB,128MB}]
                           [--spi-connection SPI_CONNECTION] [--no-progress]
                           [--verify] [--encrypt]
                           [--encrypt-files <address> <filename> [<address> <filename> ...]]
                           [--ignore-flash-encryption-efuse-setting] [--force]
                           [--compress | --no-compress]
                           <address> <filename> [<address> <filename> ...]
esptool write_flash: error: argument <address> <filename>: [Errno 2] No such file or directory: 'storage.bin'

esptool.py --chip esp32s3 -p /dev/ttyACM0 -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 80m --flash_size 2MB 0x0 bootloader/bootloader.bin 0x10000 littlefs_example.bin 0x8000 partition_table/partition-table.bin 0x110000 storage.bin
CMake Error at run_serial_tool.cmake:66 (message):
  
  /home/esp/.espressif/python_env/idf5.2_py3.11_env/bin/python;;/home/esp/dev/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32s3
  failed.



FAILED: CMakeFiles/flash /home/esp/src/github/testing/littlefs/build/CMakeFiles/flash 
cd /home/esp/dev/esp-idf/components/esptool_py && /usr/bin/cmake -D IDF_PATH=/home/esp/dev/esp-idf -D "SERIAL_TOOL=/home/esp/.espressif/python_env/idf5.2_py3.11_env/bin/python;;/home/esp/dev/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32s3" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/esp/src/github/testing/littlefs/build -P /home/esp/dev/esp-idf/components/esptool_py/run_serial_tool.cmake
ninja: build stopped: subcommand failed.
ninja failed with exit code 1, output of the command is in the /home/esp/src/github/testing/littlefs/build/log/idf_py_stderr_output_5021 and /home/esp/src/github/testing/littlefs/build/log/idf_py_stdout_output_5021
```

### Steps to reproduce.

```Bash
$ cd esp-idf/examples/storage/littlefs
$ idf.py set-target esp32s3
$ idf.py build
$ idf.py flash
```

**Comment by AxelLin** [comment]
This is already fixed by littlefs-v1.14.8, see https://github.com/joltwallet/esp_littlefs/issues/192 .

**Comment by doragasu** [comment]
Awesome, tested with v1.14.8 and I can confirm it's working, closing.

Thanks a lot for support!
